.github/workflows/ci.yml

```diff
@@ -140,7 +140,12 @@ jobs:
         run: echo $PATH
       - name: Test `nix` with `$GITHUB_PATH`
         if: success() || failure()
-        run: nix run nixpkgs#fortune
+        run: |
+          nix run nixpkgs#fortune
+          nix profile install nixpkgs#fortune
+          fortune
+          nix store gc
```

Comment:
This also makes sure we don't GC the nix package from the store.

Reply:
Nit: do we need `nix run` if we're going to `nix profile install` it?

Reply:
This is to ensure `nix store gc` doesn't clear `nix` ala #138